src/repository/mod.rs

```diff
@@ -1,7 +1,7 @@
-use std::fs::File;
 use std::io::BufReader;
 use std::path::PathBuf;
 use std::process::Command;
+use std::{fs::File, io::BufRead};
```

Comment:
Can you use the same syntax as other dependencies, i.e. only group `std::fs`  and `std::io` uses together?